presto-docs/src/main/sphinx/security/server.rst

```diff
@@ -199,19 +204,4 @@ to ``stdout`` to appear in the logs.
 
 The amount and usefulness of the information the Kerberos debugging output
 sends to the logs varies depending on where the authentication is failing.
-Exception messages and stack traces can also provide useful clues about the
-nature of the problem.
-
-.. _server_additional_resources:
-
-Additional resources
-^^^^^^^^^^^^^^^^^^^^
-
-`Common Kerberos Error Messages (A-M)
-<http://docs.oracle.com/cd/E19253-01/816-4557/trouble-6/index.html>`_
-
-`Common Kerberos Error Messages (N-Z)
-<http://docs.oracle.com/cd/E19253-01/816-4557/trouble-27/index.html>`_
-
-`MIT Kerberos Documentation: Troubleshooting
-<http://web.mit.edu/kerberos/krb5-latest/doc/admin/troubleshoot.html>`_
+Exception messages and stack traces can also provide useful clue
```

Comment:
The sentence looks unfinished. I think this is because the removal of following content was not intentional.

Reply:
+ Commit message updated.